scripts/init.sh

```diff
@@ -62,6 +62,7 @@ fi
 
 if [ ! -f "$STDOUT" ]; then
     mkdir -p $(dirname $STDOUT)
+    chown $USER:$GROUP $(dirname $STDOUT)
```

Comment:
Opps, that's true, I was trying to fix the corner case when you change from the default user influxdb to another user and the file/directory already exist.

I will revert this change.